task/task-darwinia-ethereum/src/migrate/v0.rs

```diff
@@ -16,7 +16,7 @@ fn migrate_scan_darwinia(microkv: &NamespaceMicroKV) -> anyhow::Result<()> {
     let key = "last-tracked-darwinia-block";
     let block_darwinia: Option<u64> = microkv.get_as(key)?;
     if let Some(block) = block_darwinia {
-        microkv.put("scan.darwinia.next", &format!("{}", block))?;
+        microkv.put("scan.darwinia.next", &block)?;
```

Comment:
I think currently no one upgrade to the latest version. so changing old migrate is safe.